src/util/object_factory_parameters.h

```diff
@@ -70,6 +72,14 @@ struct object_factory_parameterst
   /// Force string content to be ASCII printable characters when set to true.
   bool string_printable = false;
 
+  /// Force numerical primitive inputs to fall within the bounds
```

Comment:
The documentation should also tell what the type represents not how it will be used.